src/code/z_parameter.c

```diff
@@ -169,15 +169,18 @@ static Gfx sSetupDL_80125A60[] = {
 };
 
 // original name: "alpha_change"
-void Interface_ChangeAlpha(u16 alphaType) {
-    if (alphaType != gSaveContext.unk_13EA) {
-        osSyncPrintf("ＡＬＰＨＡーＴＹＰＥ＝%d  LAST_TIME_TYPE=%d\n", alphaType, gSaveContext.unk_13EE);
-        gSaveContext.unk_13EA = gSaveContext.unk_13E8 = alphaType;
-        gSaveContext.unk_13EC = 1;
+void Interface_SetHudVisibility(u16 hudVisibility) {
```

Comment:
I think it's a bit misleading to name this function as if it was a simple setter, since it actually sets multiple variables and most importantly it resets the timer (ie. it's not equivalent to just setting `hudVisibility`).
We could probably keep the "change" part and have it as `Interface_ChangeHudVisibility`?